# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/vm/Opcodes.h
# Commit: 07f0b26b5449
# Full Hash: 07f0b26b54495095213f80a7808cc7ef90c7454d
# Author: Jonatan Klemets <jonatan.r.klemets@gmail.com>
# Date: 2021-12-02 09:42:49
# Regressor Bug: 1736060
# File Overlap Count: 1
# Description:
#   Bug 1736060 - Part 6: Implement import assertions for dynamic imports. r=mgaudet,arai
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D126046
# ==============================================================================

diff -r d9fa28df9787 -r 07f0b26b5449 js/src/vm/Opcodes.h
--- a/js/src/vm/Opcodes.h	Wed Dec 01 18:26:10 2021 +0000
+++ b/js/src/vm/Opcodes.h	Wed Dec 01 18:26:10 2021 +0000
@@ -802,9 +802,9 @@
      *   Category: Expressions
      *   Type: Other expressions
      *   Operands:
-     *   Stack: moduleId => promise
+     *   Stack: moduleId, options => promise
      */ \
-    MACRO(DynamicImport, dynamic_import, NULL, 1, 1, 1, JOF_BYTE) \
+    MACRO(DynamicImport, dynamic_import, NULL, 1, 2, 1, JOF_BYTE) \
     /*
      * Push the `import.meta` object.
      *
